.github/workflows/subtree.yml

```
@@ -29,7 +30,26 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: nxtlvlsoftware/git-subtree-action@1.1
+      - uses: pnpm/action-setup@v2.2.4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - run: echo "link-workspace-packages=false" >> .npmrc
+
+      - name: Install dependencies (create separate lockfiles)
+        run: |
+          cp pnpm-lock.yaml ./examples/${{ matrix.path }}/pnpm-lock.yaml
```

could maybe just remove workspace and run install in the example path?

```suggestion
          rm -f pnpm-workspace.yaml pnpm-lock.yaml
```